chromadb/api/models/AsyncCollection.py

```diff
@@ -0,0 +1,333 @@
+from typing import (
+    TYPE_CHECKING,
+    Optional,
+    Union,
+)
+import numpy as np
+
+from chromadb.api.types import (
+    URI,
+    CollectionMetadata,
+    Embedding,
+    Include,
+    Metadata,
+    Document,
+    Image,
+    Where,
+    IDs,
+    GetResult,
+    QueryResult,
+    ID,
+    OneOrMany,
+    WhereDocument,
+)
+
+from .CollectionCommon import CollectionCommon
```

Comment:
i think we should not use relative imports - they can break things wierdly and would prefer we just full path

Reply:
2a3ae81d
https://github.com/chroma-core/chroma/issues/2334